src/server/routes/units.js

```diff
@@ -185,10 +187,10 @@ router.post('/addUnit', async (req, res) => {
 				);
 				await newUnit.insert(t);
 			});
-			res.sendStatus(200);
+			success(res);
 		} catch (err) {
 			log.error(`Error while inserting new unit ${err}`, err);
-			res.sendStatus(500);
+			failure(res, 500, err.toString() + ' with detail ' + err['detail']);
 		}
```

Comment:
From what I recall, this is the only message that has the err and the err detail. From the messages I could easily generate, the detail is included in the err string. Thus, I propose we remove it unless someone sees something I do not. Whatever is done, it would be nice to be consistent.